packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -3552,6 +3557,41 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     }
   );
 
+  getExpandCellTooltip = memoize(
+    (expandCellTooltipProps: CSSProperties): ReactNode => {
+      if (expandCellTooltipProps == null) {
+        return null;
+      }
+
+      const wrapperStyle: CSSProperties = {
+        position: 'absolute',
+        ...expandCellTooltipProps,
+        pointerEvents: 'none',
+      };
+
+      const popperOptions: PopperOptions = {
+        placement: 'bottom-start',
+      };
+
+      return (
+        <div style={wrapperStyle}>
+          <Tooltip
+            key={Date.now()}
+            options={popperOptions}
+            ref={this.handleTooltipRef}
+          >
+            <div style={{ textAlign: 'left' }}>
+              Click to expand row
```

Comment:
After trying it I noticed this `expand` should probably be isExpanded ? collapse : expand